Makefile

```diff
@@ -309,6 +333,17 @@ define create-local-cluster
 	--agents 1 --no-lb --k3s-server-arg "--no-deploy=traefik,servicelb,metrics-server" --network $(CLUSTER_GSLB_NETWORK)
 endef
 
+define deploy-k8gb-with-helm
+	cd chart/k8gb && helm dependency update
+	helm -n k8gb upgrade -i k8gb chart/k8gb -f $(VALUES_YAML) \
+		--set k8gb.hostAlias.enabled=true \
+		--set k8gb.hostAlias.ip="`$(call get-host-alias-ip,k3d-$1,k3d-$2)`" \
+		--set k8gb.imageTag=$3 $4 \
+		--set k8gb.log.format=$(LOG_FORMAT) \
+		--set k8gb.log.level=$(LOG_LEVEL) \
+		--wait --timeout=2m0s
```

Comment:
why exactly 2m?

Reply:
@ytsarev I supposed 2m is just enough for starting k8gb timeout. The default timeout is `5m`

Reply:
ok, let's stick with this threshold